### ISSUE TITLE
Error Linking Slack

### ISSUE THREAD

**Issue by rcastle1** [badgerloop-software/BOM-Webapp#5]
![image](https://user-images.githubusercontent.com/25836359/63229467-4c02bd00-c1b5-11e9-8933-699376a6261e.png)
I get this error when I first logged in on slack. My login still appears to have gone through as I was able to re-load finance.badgerloop.com and was logged in

**Comment by EUdds** [comment]
This is an error we're seeing due to timeouts in slack authorization. Did you take an abnormally long time to click continue?

**Comment by rcastle1** [comment]
probably less than 5s. I'm unable to reproduce by logging out and logging back in.

**Comment by EUdds** [comment]
Yeah, I haven't been able to purposefully reproduce this yet. If it does come up again can you detail everything you were doing?

**Comment by rcastle1** [comment]
I cannot replicate the couple of times I've logged in.
